### PR TITLE
Add resource viewer to workload

### PR DESCRIPTION
**Pull request by bryanl**
re: #351

Screen Shot 2020-01-19 at 5 57 51 PM

Signed-off-by: bryanl <bryanliles@gmail.com>

### CODE REVIEW

File: web/src/app/modules/overview/services/navigator/navigator.service.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  private source = new BehaviorSubject<NavigatorStatus>(null);
+  status = this.source.asObservable();
+
+  constructor() {}
+
+  addHistory(path: string) {
+    console.log(
+      `tail: ${
+        this.internalHistory[this.internalHistory.length - 1]
+      }; path: ${path}`
```
Comment (GuessWhoSamFoo) [inline web/src/app/modules/overview/services/navigator/navigator.service.ts]: URLs with a query param (such as clicking on tabs) will be encoded and causes navigator to go to an invalid path. A filter might be needed.

File: web/src/app/modules/overview/components/presentation/title/title.component.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/*!
```
Comment (GuessWhoSamFoo) [inline web/src/app/modules/overview/components/presentation/title/title.component.scss]: nit: !

File: web/src/app/modules/overview/components/presentation/navigator/navigator.component.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+} from '@angular/core';
+import { NavigatorService } from '../../../services/navigator/navigator.service';
+
+@Component({
+  selector: 'app-navigator',
+  templateUrl: './navigator.component.html',
+  styleUrls: ['./navigator.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NavigatorComponent implements OnInit {
```
Comment (GuessWhoSamFoo) [inline web/src/app/modules/overview/components/presentation/navigator/navigator.component.ts]: onDestroy should detach the `ChangeDetectorRef` and possibly unsubscribe

**Comment by bryanl** [comment]
@GuessWhoSamFoo I removed the navigator. It isn't ready for prime time.

**Comment by GuessWhoSamFoo** [comment]
Screenshot_20200122_143907

z-axis for span tag needs to be higher. Not blocking

**Comment by bryanl** [comment]
@GuessWhoSamFoo I'm going to punt on this one. We need to replace the tooltips library as all it does is make problems.

**Review by GuessWhoSamFoo — APPROVED** [review]
This looks fine. Errors relating to #549 can be handled in a future release

**Pull request merged**